wordpress-mobile/WordPress-Android
# [Site Name] Removes Site Intent Question A/B test

Fixes #16304

To test:
1. Start the site creation flow
2. Verify that the Site Intent Screen appears

## Regression Notes
1. Potential unintended areas of impact
N/A

3. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

4. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
